guides/v2.1/config-guide/cli/config-cli-subcommands-i18n.md

```diff
@@ -15,30 +15,36 @@ Magento translations enable you to customize and localize your store for multipl
 
 This topic discusses how to generate:
 
--   Translation dictionaries, which are a convenient way to translate *some* words and phrases, such as those for a custom {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}module{% endglossarytooltip %} or {% glossarytooltip d2093e4a-2b71-48a3-99b7-b32af7158019 %}theme{% endglossarytooltip %}.
+-   Translation dictionaries, which are a convenient way to customize or translate *some* words and phrases, such as those for a custom {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}module{% endglossarytooltip %} or {% glossarytooltip d2093e4a-2b71-48a3-99b7-b32af7158019 %}theme{% endglossarytooltip %}.
 -   Language packages, which enable you to translate *any or all* words and phrases in the Magento application.
 
-For more information, see [Translation]({{ page.baseurl }}/frontend-dev-guide/translations/xlate.html).
+For more information, see [Translations overview].
+
+We also accept [Community Engineering contributions] using CrowdIn for translations. We plan to provide translation packs through the project. Join us and other contributors around the globe working to localize Magento.
 
 ## Generate a translation dictionary {#config-cli-subcommands-xlate-dict}
 
-You can generate a translation dictionary to use by itself (for example, to translate words and phrases in a custom module) or for use by a {% glossarytooltip 9c4c7b9b-43f0-4454-8e8c-fb62ad40c35f %}language package{% endglossarytooltip %}. See one of the following sections:
+You can generate a translation dictionary to customize existing strings, translate words and phrases in a custom module, localize a theme, or for a {% glossarytooltip 9c4c7b9b-43f0-4454-8e8c-fb62ad40c35f %}language package{% endglossarytooltip %}. 
+
+See one of the following sections:
```

Comment:
Why?

Reply:
cleaned up and removed the links, added them in a different way